src/ui-fragment/src/utils.tsx

```diff
@@ -111,6 +112,8 @@ export const renderComponents = (state: any, setState: (state: any) => void, set
 		case 'text':
 			return <UIText key={state.id} state={state} setState={setState} setGlobalState={setGlobalState} />;
 
+		case 'tooltip':
+			return <UITooltip key={state.id} state={state} setState={setState} setGlobalState={setGlobalState} />;
```

Comment:
```suggestion
		case 'tooltip':
			return <UITooltip key={state.id} state={state} setState={setState} setGlobalState={setGlobalState} />;

```

and it would make sense if it was after text-input, or tile, no?
same with the import

_It does seem a little bit arbitrary with the overflow-menu interjecting like this 🤔_

Reply:
Updated